### ISSUE TITLE
yumex-dnf: misc.py:114:rgb_to_hex:TypeError: %X format: an integer is required, not float

### ISSUE THREAD

**Issue by raveit65** [timlau/yumex-dnf#93]
Forwarded from upstream report:
https://bugzilla.redhat.com/show_bug.cgi?id=1316624
stacktrace:
https://bugzilla.redhat.com/attachment.cgi?id=1134940

```
[rave@mother ~]$ yumex-dnf
/usr/lib/python3.5/site-packages/yumex/__init__.py:69: Warning: invalid cast from 'GtkBox' to 'GtkWindow'
  self.ui.add_from_file(const.DATA_DIR + '/yumex.ui')

(yumex-dnf:17751): Gtk-CRITICAL **: gtk_window_add_accel_group: assertion 'GTK_IS_WINDOW (window)' failed

(yumex-dnf:17751): Gtk-WARNING **: State 0 for YumexPackageView 0x55716b826bd0 doesn't match state 128 set via gtk_style_context_set_state ()
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/yumex/__init__.py", line 1329, in do_command_line
    self.do_activate()
  File "/usr/lib/python3.5/site-packages/yumex/__init__.py", line 1232, in do_activate
    self.win = YumexWindow(self, self.status)
  File "/usr/lib/python3.5/site-packages/yumex/__init__.py", line 401, in __init__
    CONFIG.conf.color_normal = color_to_hex(color_normal)
  File "/usr/lib/python3.5/site-packages/yumex/misc.py", line 118, in color_to_hex
    return rgb_to_hex(color.red, color.green, color.blue)
  File "/usr/lib/python3.5/site-packages/yumex/misc.py", line 114, in rgb_to_hex
    return "#%02X%02X%02X" % (r, g, b)
TypeError: %X format: an integer is required, not float
```

Since good old yumex is droped in f24, yumex-dnf is the only package-manager for several fedora spins, ie. Mate, Cinnamon and others, and we're several weeks before alpha release.
Can you please take a look into it?
Btw. the gtk_style_context_set_state warning can easily fixed like this.
https://github.com/compiz-reloaded/emerald/commit/6436f0b9df783b1f2922932b35c4f5140f49033a
Well, this isn't python but it should work in the same way.
See this gtk+ blog for references:
https://blogs.gnome.org/mclasen/2015/11/20/a-gtk-update/

**Comment by genodeftest** [comment]
This is a duplicate of #81 which has already been closed in the 4.1.6 release.

**Comment by raveit65** [comment]
Ok, yumex-dnf is at bodhi for f24 and it starts again.
But honestly, it spams really my terminal.

```
[rave@mother ~]$ yumex-dnf 
/usr/lib/python3.5/site-packages/yumex/__init__.py:69: Warning: invalid cast from 'GtkBox' to 'GtkWindow'
  self.ui.add_from_file(const.DATA_DIR + '/yumex.ui')

(yumex-dnf:15160): Gtk-CRITICAL **: gtk_window_add_accel_group: assertion 'GTK_IS_WINDOW (window)' failed
/usr/lib/python3.5/site-packages/dnfdaemon/client/__init__.py:291: PyGIDeprecationWarning: GObject.G_MAXINT is deprecated; use GLib.MAXINT instead
  user_data=data, timeout=GObject.G_MAXINT)
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/yumex/gui/views.py", line 1124, in on_mouse_motion
    tags = widget.get_iter_at_location(x, y).get_tags()
AttributeError: '_ResultTuple' object has no attribute 'get_tags'
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/yumex/gui/views.py", line 1124, in on_mouse_motion
    tags = widget.get_iter_at_location(x, y).get_tags()
AttributeError: '_ResultTuple' object has no attribute 'get_tags'
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/yumex/gui/views.py", line 1124, in on_mouse_motion
    tags = widget.get_iter_at_location(x, y).get_tags()
AttributeError: '_ResultTuple' object has no attribute 'get_tags'
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/yumex/gui/views.py", line 1124, in on_mouse_motion
    tags = widget.get_iter_at_location(x, y).get_tags()
AttributeError: '_ResultTuple' object has no attribute 'get_tags'


etc.
```

Anyways, bug is fixed.
